hands_on/main/ball.script

```diff
@@ -4,7 +4,7 @@ local score = 0
 function on_message(self, message_id, message, sender)
 	if message_id == hash("collision_response") and message.group == hash("pin") then
 		go.delete(message.other_id)
-		score = score + 1
+		score = score + 35
```

Comment:
Why this change?

Reply:
I made the score a higher number while adjusting the label component to make sure there was enough room while the score incremented during playtime. Good catch I should have reverted back to 1 afterward.